api/app/signals/apps/sigmax/outgoing.py

```diff
@@ -35,6 +35,20 @@ class SigmaxException(Exception):
 
 # TODO SIG-593: implement data mapping if and when it is defined
 
+def _generate_omschrijving(signal):
+    """Generate brief descriptive text for list view in CityControl"""
+    # Note: we do not mention main or subcategory here (too many characters)
+    is_urgent = 'JA' if signal.priority.priority == Priority.PRIORITY_HIGH else 'NEE'
+    stadsdeel = signal.location.stadsdeel if signal.location.stadsdeel else '-'
```

Comment:
You can write this down as: `signal.location.stadsdeel or '_'`.
Syntactic sugar.